openapi-diff/src/modeler/AutoRest.Swagger.Tests/Resource/Swagger/old/removed_parameter.json

```diff
@@ -0,0 +1,55 @@
+﻿{
+  "swagger": 2.0,
+  "info": {
+    "title": "removed_parameter",
+    "version": "1.0"
+  },
+  "host": "localhost:8000",
+  "schemes": [ "http", "https" ],
+  "paths": {
+    "/api/Parameters/{a}": {
+      "get": {
+        "tag": [ "Parameters" ],
+        "operationId": "Parameters_Get",
+        "produces": [
+          "text/plain"
+        ],
+        "parameters": [
```

Comment:
@mikekistler per the test I added, here the removal of the params named ` "path_param", "path_param_req", "query_param_req", "body_param_req" ` will result in `RemovedRequiredParameter` breaking chane each, while removal of `"body_param", "query_param"` will result in `RemovedOptionalParameter` breaking change.

Notably, no `ChangedParameterOrder` breaking change will be reported.